xemc3/data/default.yaml

```
@@ -131,23 +143,45 @@ fort.4:
 fort.40:
   type: mapped
   vars:
-    fort.40_%d: {}
+    SOURCE_P_%d:
+      units: m$^{-3}$
+      scale: 1000000.0
+      description: Particles sources normalised by the recycling flux
 fort.42:
   type: mapped
   vars:
-    fort.42_%d: {}
+    SOURCE_E_E:
+      units: W.m$^{-3}$
+      scale: 1000000.0
+      long_name: Energy sources for electrons due to ionisation of neutrals
 fort.43:
   type: mapped
   vars:
-    fort.43_%d: {}
+    SOURCE_E_I:
+      units: W.m$^{-3}$
+      scale: 1000000.0
+      long_name: Energy sources for ions due to ionisation of neutrals
+fort.44:
+  type: mapped
+  vars:
+    SOURCE_RAD_E:
+      units: W.m$^{-3}$
+      scale: 1000000.0
+      long_name: Energy sources for electrons from impurities
 fort.46:
   type: mapped
   vars:
-    fort.46_%d: {}
+    ALFA_SOURCE:
+      units: A.g.s$^{-1}$.m$^{-2}$
+      scale: 10000.0
```

I think it would be better to switch to particle based, rather then Ampere.
So probably 
```suggestion
      units: "g s$^{-1}$ m$^{-2}$"
      scale: 6.242e22
```
Maybe we should add more SF ... or use `A` in the scale, so we only need to set it once?